Mathlib/Topology/Category/Profinite/CofilteredLimit.lean

```diff
@@ -45,9 +49,10 @@ a clopen set in one of the terms in the limit.
 -/
 theorem exists_clopen_of_cofiltered {U : Set C.pt} (hC : IsLimit C) (hU : IsClopen U) :
     ∃ (j : J) (V : Set (F.obj j)) (_ : IsClopen V), U = C.π.app j ⁻¹' V := by
+  have := preserves_smaller_limits_toTopCat.{u, v}
   -- First, we have the topological basis of the cofiltered limit obtained by pulling back
   -- clopen sets from the factors in the limit. By continuity, all such sets are again clopen.
-  have hB := TopCat.isTopologicalBasis_cofiltered_limit.{u, u} (F ⋙ Profinite.toTopCat)
+  have hB := TopCat.isTopologicalBasis_cofiltered_limit.{u, v} (F ⋙ Profinite.toTopCat)
```

Comment:
Sorry, totally missed the review request. The PR looks good to me.

I tried a bit to switch to UnivLE, and indeed to fix this we need something like `homeoShrink` in [TopCat.limitConeInfi](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Category/TopCat/Limits/Basic.html#TopCat.limitConeInfi). For different categories we need to put multitudes of structures on `Shrink`, which is not ideal but doable (see how we added tons of algebraic and order instances on type synonyms). I think I'd like to go further with my idea of shrinking the indexing category `J` rather than the limit object in the category, and I'd start with refactoring [CategoryTheory.Limits.Types.Small.limitCone](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Limits/Types.html#CategoryTheory.Limits.Types.Small.limitCone). I think with proper APIs (maybe utilizing EqToHom), we can avoid descending into dependent hell.

Reply:
Your suggested change to `hB` is already there in the latest commit. Maybe you were reviewing an older commit?

Reply:
Never mind, I read the red and green above as you suggesting a change but it's actually my change.